luda-editor/luda-editor-client/src/editor/clip_editor/camera_clip_editor/wysiwyg_editor/mod.rs

```diff
@@ -28,26 +28,8 @@ impl WysiwygEditor {
             height: props.xywh.height,
         };
 
-        let camera_angle = &match &props.job {
```

Comment:
moved this to camera_clip_editor because preview also need this job-pre-executed camera_angle for preview.